core/src/test/java/org/elasticsearch/monitor/jvm/JvmInfoTests.java

```diff
@@ -58,4 +62,38 @@ private boolean flagIsEnabled(String argline, String flag) {
         return argline.charAt(index - 1) == '+';
     }
 
+    public void testSerialization() throws IOException {
+        JvmInfo jvmInfo = JvmInfo.jvmInfo();
```

Comment:
looking again, I am not sure it makes sense to spend time on randomizing all those values. That could take this test very far from what happens in reality with this class, as values are taken from the running jvm. then this test could be removed as the same is already done in NodeInfoStreamingTests, which should cover for all the changes I made.